.circleci/workflows.yml

```diff
@@ -47,7 +56,13 @@ parameters:
   targets--x86_64-unknown-linux-gnu--self-test:
     type: string
     default: ""
-  targets--x86_64-unknown-linux-gnu--std-only:
+  targets--x86_64-unknown-linux-gnu--std:
```

Comment:
Why don't we need `targets--aarch64-unknown-linux-gnu--std` as well?

Reply:
Because cross-compilation targets can be built on just a single builder, so we don't need to also build them on aarch64 Linux.